# Autostart polkit-gnome-authentication-agent-1 to enable polkit support
# This should be changed when LXDE will have a polkit-agent.
Index: lxde-common/autostart
===================================================================
--- lxde-common.orig/autostart	2010-05-25 22:58:28.000000000 +0200
+++ lxde-common/autostart	2010-05-25 22:59:13.000000000 +0200
@@ -1,3 +1,4 @@
 @xscreensaver -no-splash
 @lxpanel --profile LXDE
 @pcmanfm --desktop --profile LXDE
+@/usr/libexec/polkit-gnome-authentication-agent-1
